File: Unidad_3/Java/Programa17_U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa17_u3;
import java.util.Scanner;
/**
 *
 * @author LaboratorioLTI2
 */

public class Programa17_U3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner (System.in);       
        char opcion;
        do{
                    System.out.println("Bienvenido a una sola ejecucion");
                    System.out.println("¡¡Escribe n para salir!!!");
                    opcion=sc.next().charAt(0);
        }while(opcion !='n');       
    }
}

File: Unidad_3/C#/Programa17_U3.cs
namespace Programa17_U3
{
    class Program
    {
        static void Main(string[] args)
        {
            char opcion;
            do
            {
                Console.WriteLine("Bienvenido a una sola ejecución");
                Console.WriteLine("¡¡Escribe n para salir!!!");
                opcion = Console.ReadKey().KeyChar;
            } while (opcion != 'n');
        }
    }
}